support/source/ios/build.sh

```diff
@@ -7,7 +7,7 @@ echo '======================================================='
 echo ''
 
 FRAMEWORK_NAME="Mockifer"
-PRODUCT_PATH_ROOT="./../../../products/libs/ios"
+PRODUCT_PATH_ROOT="./../../../Automation/MockiferEcobee/FrameworkBuild"
```

Comment:
The changes in this shell script appear to be specific to your own build environment? Suggest reverting the shell script changes.